backend/backend_app/api/models.py

```diff
@@ -0,0 +1,51 @@
+from lxml import etree
+from ..db import db
+
+
+class SimpleParser(object):
+    def __init__(self, xml):
+        self.namespaces = {'n': 'urn:hl7-org:v3'}
+        self.tree = etree.fromstring(xml)
+
+    def first_name(self):
+        found = self.tree.xpath(
+            "//n:patient/n:name/n:given/text()",
+            namespaces=self.namespaces)
+        if found:
+            return found[0]
```

Comment:
Nice! I gotta add this behavior to the PDMP face so lookups don't fail if an element is absent